lib/class-wp-theme-json-gutenberg.php

```diff
@@ -1119,13 +1126,41 @@ public function merge( $incoming, $update_or_remove = 'remove' ) {
 			}
 			foreach ( $to_append as $path_to_append ) {
 				$path          = array_merge( $metadata['path'], $path_to_append );
-				$incoming_node = _wp_array_get( $incoming_data, $path, array() );
+				$incoming_node = _wp_array_get( $incoming_data, $path, null );
```

Comment:
I think we can simply remove the 3rd argument here... `$default` is already `null` in the function.

```suggestion
				$incoming_node = _wp_array_get( $incoming_data, $path );
```